Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -1637,6 +1637,28 @@ def visit_CClassDefNode(self, node):
                 self._inject_pickle_methods(node)
         return node
 
+    def _inject_fusedcpdef_sigindex(self, node):
+        print(node)
+        sigindex_dict = TreeFragment(
+            u"""            
+            cdef dict %(fused_cpdef_globalindex)s = {}
+            
+            %(fused_cpdef_globalindex)s = {}
```

Comment:
@scoder @robertwb You're probably the ones most familiar with Cython internals, and I think trying to reverse-engineer the best place for this could easily take quite a very disproportionate amount time for me at this point. Where should the code generation be placed for a statically typed global dictionary in any modules that use `fused` `cpdef`s?

Reply:
It's actually not so global, give that it will eventually be specific to an instance of the module (think of reloading and subinterpreters), and even to the function, really. I think a mutable default argument captures this best, after all